style/components/profile-card.css

```diff
@@ -0,0 +1,49 @@
+.profile-card {
+  padding: 25px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 15px;
+  max-width: 420px;
+}
+
+.profile-card__img {
+  width: 100%;
+  height: 100%;
+  border-radius: 999px;
+}
+
+.profile-card__name {
+  color: var(--c-primary);
+  font-size: 1.5rem;
+  font-weight: bold;
+  justify-self: center;
+  align-self: center;
+}
+
+.profile-card__description,
+.profile-card__skill-header,
+.tag-list,
+.profile-card__logout-btn {
```

Comment:
This is quite a list of unrelated things to get this property. Please consider a utility class to give them.